tutorial/123_pea2_en.ipynb

```diff
@@ -115,13 +115,13 @@
     "\n",
     "In this case, diagonal elements of $D$ are eigenvalues of $\\hat H$ and each column of $P$ are eigenvalues of $\\hat H$. Now, we're considering a 2x2 Hermitian with eigenvalues summing up to zero, so, elements of $D$ are $\\pm E$ when $\\hat H$ has an eigenvalue $E$.\n",
     "\n",
-    "Thus, we generate random real number $E$ and make $D$, then, generate random three real numbers $\\theta, \\phi, \\lambda$ and make $P$ using U3 gate, i.e. $P = \\mathrm{U3}(\\theta, \\phi, \\lambda)$.\n",
+    "Thus, we generate random real number $E$ and make $D$, then, generate random three real numbers $\\theta, \\phi, \\lambda$ and make $P$ using U gate, i.e. $P = \\mathrm{U}(\\theta, \\phi, \\lambda)$. Here, we ignore the parameter $\\gamma$ of U gate because it is irrelevant to eigenvalues.\n",
```

Comment:
is the last sentence OK?

Reply:
I think OK.
As a note, in the use of CU gate, the parameter gamma changes relative phase and could be unignorable.
In this tutorial, there are no problem.